xl_auth/commands.py

```diff
@@ -102,8 +102,7 @@ def clean():
 @click.command()
 @click.option('-e', '--email', required=True, default=None, help='Email for user')
 @click.option('-n', '--full-name', default=None, help='Full name for user (default: None)')
-@click.option('-p', '--password', default='password',
-              help='Password for user (default: "password")')
+@click.option('-p', '--password', default=None, help='Password for user (default: None)')
```

Comment:
How does this work? We actually set the password to None or do we generate one? If it's the latter, maybe we should change `None` to `auto-generated` in the help text?

Reply:
Yes, we just pass in some urandom output to bcrypt hasher, to give it some value with the right number of chars (and not have to deal with NOT NULL IntegrityErrors). I think "auto-generated" is a bit misguiding.. When autogenerating passwords, you kind of expect that the tool you're interacting with tells you what it made up for password. Now it's not, so for all practical purposes there's no password.

When creating a user you get this:
```
(py36venv) EMB2:xl_auth mblomdahl$ flask create_user -e mats@kb.se --is-admin --is-active
Created account with login mats@kb.se:None
```

If's saying it's auto-generated, I'd say it's more intuitive if's the tool behaves like this:
```
(py36venv) EMB2:xl_auth mblomdahl$ flask create_user -e mats@kb.se --is-admin --is-active
Created account with login mats@kb.se:tike2!Palmas
```

With some nice auto-generated memorable passwords, as generated by keychain for example;
![screen shot 2017-11-23 at 9 40 25 am](https://user-images.githubusercontent.com/786326/33164315-78d48dfc-d032-11e7-97ff-b12a073a60a3.png)